### PR TITLE
manager.c: Add missing parameters to Login documentation

### PR DESCRIPTION
**Pull request by gtjoseph**
* Added the AuthType and Key parameters for MD5 authentication.

* Added the Events parameter.

Resolves: #689

### CODE REVIEW

**Comment by gtjoseph** [comment]
cherry-pick-to: 16
cherry-pick-to: 18
cherry-pick-to: 20
cherry-pick-to: 21
cherry-pick-to: certified/18.9
cherry-pick-to: certified/20.7

**Comment by gtjoseph** [comment]
N.B.  We still produce the documentation for the 16 branch even though we don't accept changes to it.

File: main/manager.c [hunk 1]
```diff
@@ -165,14 +165,32 @@
...
+				</enumlist>
+			</parameter>
 			<parameter name="Secret">
-				<para>Secret to login with as specified in manager.conf.</para>
+				<para>Plain text secret to login with as specified in manager.conf.</para>
+			</parameter>
+			<parameter name="Key">
+				<para>Key to use with MD5 authentication.  To create the key, you must
+				first call the <literal>Challenge</literal> action, create an MD5 hash
+				with the result, then update the hash with the secret as specified
```
Comment (seanbright) [inline main/manager.c:181]: What needs to be hashed specifically? The challenge response concatenated with the plain text password?

Comment (gtjoseph) [reply main/manager.c:181]: Just as I described.  You need to create the hash first with the challenge response, then update the hash with with the secret.  Exactly how you do it depends on the technology you're using to interface with AMI.

If you're using python for instance...
```
import hashlib
m = hashlib.md5()
m.update(response_from_challenge)
m.update(your_secret)
key = m.hexdigest()
## '031edd7d41651593c5fe5c006fa5752b'
```

Do I need to include the example?

Comment (gtjoseph) [reply main/manager.c]: Added python example to the description.

**Review by seanbright — APPROVED** [review]

**Review by jcolp — CHANGES_REQUESTED** [review]
The commit message has "parameteers" instead of "parameters".

**Review by jcolp — APPROVED** [review]

**Pull request merged**